src/gui/utils.cpp

```diff
@@ -76,35 +75,6 @@ qreal Utils::Gui::screenScalingFactor(const QWidget *widget)
 #endif // Q_OS_WIN
 }
 
-QPixmap Utils::Gui::scaledPixmap(const QIcon &icon, const QWidget *widget, const int height)
```

Comment:
There is also a second consideration to be discussed, is it correct/preferable that a theme manager needs to be concerned about image scaling? Sure it simplifies the code using it, but the big picture looks weird to me. Of course we can have helper functions that provide such conveniences but they should be static functions of the theme manager class.